airbyte-integrations/connectors/source-monday/source_monday/manifest.yaml

```diff
@@ -74,6 +74,7 @@ definitions:
     $parameters:
       name: "boards"
       path: ""
+      items_per_page: 100
```

Comment:
The current Pagination already uses 100 items check `default_paginator` component. Are you able to build the dev version and test it?

Reply:
I have run `python main.py read --config secrets/config.json --catalog configured_stream.json `. It works for us with 200+ boards. 
This is not only about pagination, but also about `limit` parameter; 
https://github.com/airbytehq/airbyte/blob/44fa2844ac4277ffc1e66762b6e5133ab6b8fef8/airbyte-integrations/connectors/source-monday/source_monday/manifest.yaml#L22

Right now the logic is following:
1. `limit`  becomes None in [here](https://github.com/airbytehq/airbyte/blob/44fa2844ac4277ffc1e66762b6e5133ab6b8fef8/airbyte-integrations/connectors/source-monday/source_monday/graphql_requester.py#L98) 
2. Because `limit` is not specified, Monday returns default 25 rows for `boards`.
3. Pagination checks if it needs to visit another page with the condition: `len(last_records) < self.page_size`. And it gets `25 < 100`. So there is no query for the second page.

This PR solves the issue: the limit is set and pagination starts working when needed. Although it does not look self-explaining

Reply:
```
(venv) igsaf@igsaf-nix:~/IdeaProjects/airbyte/airbyte-integrations/connectors/source-monday (fix-monday-boards) $ python main.py read --config secrets/config.json --catalog configured_stream.json | tail -n 4 
{"type": "LOG", "log": {"level": "INFO", "message": "Read 221 records from boards stream"}}
{"type": "LOG", "log": {"level": "INFO", "message": "Finished syncing boards"}}
{"type": "LOG", "log": {"level": "INFO", "message": "SourceMonday runtimes:\nSyncing stream boards 0:00:21.135193"}}
{"type": "LOG", "log": {"level": "INFO", "message": "Finished syncing SourceMonday"}}
```